crates/router/src/core/admin.rs

```diff
@@ -1732,6 +1732,10 @@ pub(crate) fn validate_auth_and_metadata_type(
     use crate::connector::*;
 
     match connector_name {
+        // api_enums::Connector::Mifinity => {
+        //     mifinity::transformers::MifinityAuthType::try_from(val)?;
+        //     Ok(())
+        // } Added as template code for future usage
```

Comment:
This is not required to be commented, can you confirm from past template code PRs

Reply:
It is commented in template code PRs.